### PR TITLE
Replace target with restate-data and update references based on v0.8.1

### PR DESCRIPTION
**Pull request by tillrohrmann**
Replace target with restate-data and update references based on v0.8.1. This PR requires https://github.com/restatedev/restate/pull/1278 to be merged first.

### CODE REVIEW

File: docs/references/errors.md [hunk 1]
```diff
@@ -122,13 +143,13 @@ Suggestions:
 * Check for bugs in your code. Most likely no message was sent to Restate because your code is blocked and/or reached a deadlock.
 * If your code is supposed to not send any message to Restate for longer than the configured timeout, because for example is doing a blocking operation that takes a long time, change the configuration accordingly.
 
-## RT0002
+## RT0002 {#RT0002}
 
 Cannot start Restate because the configuration cannot be parsed. Check the configuration file and the environment variables provided.
 
-For a complete list of configuration options, and a sample configuration, check the [configuration docs](/operate/configuration).
+For a complete list of configuration options, and a sample configuration, check https://docs.restate.dev/restate/configuration
```
Comment (gvdongen) [inline docs/references/errors.md:150]: Replace by `the [configuration docs](/operate/configuration)`

File: docs/references/errors.md [hunk 2]
```diff
@@ -170,12 +191,33 @@ This is a generic error which can be caused by many reasons, including:
...
-## RT0007
+## RT0007 {#RT0007}
 
 A retry-able error was received from the service deployment while processing the invocation.
 
 Suggestions:
 
 * Check the service/deployment logs to get more info about the error cause, like the stacktrace.
-* Look at the [error handling docs](/develop/sdk/error-handling) for more info about error handling in services.
+* Look at the https://docs.restate.dev/services/sdk/error-handling for more info about error handling in services.
```
Comment (gvdongen) [inline docs/references/errors.md:201]: Replace by `[error handling docs](/develop/sdk/error-handling)`

File: static/schemas/config_schema.json [hunk 3]
```diff
@@ -126,12 +149,36 @@
       },
       "$ref": "#/definitions/AdminOptions"
     },
-    "tokio_runtime": {
+    "bifrost": {
```
Comment (gvdongen) [inline static/schemas/config_schema.json:152]: Is Bifrost an internal term or will we also use this towards users?

**Review by gvdongen — APPROVED** [review]
LGTM. I found a few links that will break in the next docs release. Maybe we can replace these as well already?

File: static/schemas/config_schema.json (hunk 3 above)
Comment (tillrohrmann) [reply static/schemas/config_schema.json:152]: This has not been decided yet. Atm, it is called bifrost in the configuration and that's why the generated specification is like this. Atm, the users won't have to configure it.

File: docs/references/errors.md [hunk 4]
```diff
@@ -87,32 +87,53 @@ When implementing a new service revision, make sure that:
 * The Protobuf contract and message definitions are backward compatible.
   * The new revision must implement all the methods of the previous revisions.
 
-See the [versioning documentation](/operate/versioning) for more information.
+See the [versioning documentation](https://docs.restate.dev/services/upgrades-removal) for more information.
```
[the remark this answers is not in the document]
Comment (tillrohrmann) [reply docs/references/errors.md:90]: Same here.

File: docs/references/errors.md [hunk 5]
```diff
@@ -87,32 +87,53 @@ When implementing a new service revision, make sure that:
...
+## META0009 {#META0009}
 
 The provided subscription is invalid. Subscriptions should have:
 
 * A `source` field in the format of `kafka://<CLUSTER_NAME>/<TOPIC_NAME>`. When registering, the Kafka cluster should be configured in the Restate configuration.
 * A `sink` field in the format of `service://<SERVICE_NAME>/<METHOD_NAME>`. When registering, service and method should have been previously registered as well.
 * Additional constraints may apply depending on the sink service contract
 
-Please look at the [Kafka documentation page](/develop/sdk/kafka) for more details on subscriptions and event handlers.
+Please look at the [Kafka documentation page](https://docs.restate.dev/services/sdk/kafka) for more details on subscriptions and event handlers.
```
[the remark this answers is not in the document]
Comment (tillrohrmann) [reply docs/references/errors.md:113]: And here.

File: docs/references/errors.md (hunk 2 above)
Comment (tillrohrmann) [reply docs/references/errors.md:201]: Will correct it in the Restate repo.

**Comment by tillrohrmann** [comment]
Closing this PR since we decided to not backport the storage directory change to the `release-0.8`. I will open another PR which we need to merge once the next `0.9` release gets released since this version will have the changed default storage directory.

**Pull request closed without merging**